src/app/services/Document.service.ts

```diff
@@ -12,10 +12,14 @@ export class DocumentService {
     @inject(TYPES.DOCUMENT_MAP) private mapper: DocumentMap,
   ) {}
 
-  async getAll(page = 0, pageSize = 10): Promise<IAllDocumentsOutgoingDTO> {
+  async getAll(
+    sourcesOfInterest: string[],
```

Comment:
I suggest using an object for filters criteria to avoid changing the methods signatures in the future if there is a need to add more filters